src/scripts/settings/webSettings.js

```diff
@@ -117,3 +117,7 @@ export function getPlugins() {
         return DefaultConfig.plugins;
     });
 }
+
+export function getPluginsPreLoaded() {
+    if (data) return data.plugins;
```

Comment:
No point in doing this when `getConfig` already handles caching.

Reply:
Moot due to plugin solution being discarded